frontend/modules/model/model.sagas.ts

```diff
@@ -101,14 +108,14 @@ export function* uploadModelFile({ teamspace, project, modelId, fileData }) {
 
 		const { data: { status }, data } = yield API.uploadModelFile(teamspace, modelId, formData);
 
-		if (status === uploadFileStatuses.OK) {
+		if (status === uploadFileStatuses.ok) {
```

Comment:
I don't think this ever runs. I don't see a snackbar upon upload API request.

I think the status it returns is "upload", if it errored it'll always end up in the catch clause.